client/src/components/FlexTable.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import "./FlexTable.css";
+import Icon from "./Icon";
+
+const FlexTable = ({ titleData = [], data = [], iconClick, tableId }) => {
+  return (
```

Comment:
I would say to split this component a bit. It is a bit difficult to read. Maybe you could create different smaller components like TableHeader, or TableCell

Reply:
I would also say the same. It is better to divide into as your suggestion. Thanks.